docs/PID tuning.md

```diff
@@ -29,6 +29,24 @@ changing slowly (so the P and I terms aren't having enough impact on reaching th
 in the correction in order to reach the target sooner. If the error is rapidly converging to zero, the D term causes the
 strength of the correction to be backed off in order to avoid overshooting the target.
 
+##TPA and TPA Breakpoint
+
+* Note that TPA is set via CLI or on the PID TUNING tab of the GUI.  tpa_breakpoint is set via CLI
+
+TPA applies a 'P' value reduction in relation to full Throttle. It is used to 'soften' the P value at extream throttle which sometimes induces oscellation in the aircraft.
```

Comment:
OK that's fair. I'll try to write paragraph about math tomorrow. 

Main problem is that MW PID (and it's modified versions) is mostly composed of small hacks that did solve same particular problem. The result is that the PID 'feels' right, but is hard to understand (at least for me).